### PR TITLE
Coverage increase: added tests for local storage tests

### PR DESCRIPTION
**Pull request by vishwajeetraj11**

### CODE REVIEW

**Comment by souvikmishra** [comment]
maybe remove `recieverRewardsStore` tests since we don't use that anywhere as of now @vishwajeetraj11

**Comment by paul4702** [comment]
image

@vishwajeetraj11  Can we exclude .ts files from the tests, as they are being tested even though there are no test cases written for them? I think this would further increase the test coverage.

**Comment by vishwajeetraj11** [comment]
@paul4702  those files actually have code that needs testing.

**Review by paul4702 — APPROVED** [review]

File: vite.config.ts [hunk 1]
```diff
@@ -43,11 +43,14 @@ export default defineConfig({
...
 			'src/routes/**',
 			'src/**/*.svelte',
+			'svelte.config.js',
 			'src/lib/controllers/**',
 			'src/lib/utils/services/**',
 			'src/lib/utils/constants/**',
 			'src/lib/utils/abis/**',
-			'svelte.config.js'
+			'.cache-synpress/',
+			'src/lib/data-stores/receiverRewardsStore.ts',
```
Comment (souvikmishra) [inline vite.config.ts:52]: maybe update this branch with dev, we might not need to add this here, since this file is deleted

Comment (vishwajeetraj11) [reply vite.config.ts]: done

**Review by souvikmishra — APPROVED** [review]

**Pull request merged**